cmd/delete.go

```diff
@@ -110,5 +111,6 @@ func deleteResource(args []string) (*http.Response, error) {
 	}
 	defer resp.Body.Close()
 
+	aliases.DeleteAliasesById(args[len(args)-1], resource.JsonApiType)
```

Comment:
Can you also modify `cmd/delete-all.go`, in that case you can just blow away all aliases for the json type.

Reply:
deleteResource is being called by both delete and delete-all, it s the reason I put that in deleteResource.

Reply:
Ah, I don't think that's the best place to put it for delete-all, the main reason being is that in delete-all we know we want to get rid of everything. If you delete-all a thousand resources, you will need to rewrite the file 10,000 times, and there is a mutex on the file so it will slow down the process.

Reply:
you are right